pkg/images/images_test.go

```diff
@@ -0,0 +1,137 @@
+package images
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var testDefaultImages Images
```

Comment:
The type `Images` doesn't seem to exist in this package. See the CI error:

https://drone-pr.rancher.io/rancher/rke2/2021/1/2